src/anomalib/data/folder_3d.py

```diff
@@ -292,8 +294,10 @@ class Folder3D(AnomalibDataModule):
             during validation.
             Defaults to None.
         test_split_mode (TestSplitMode): Setting that determines how the testing subset is obtained.
+        test_synthetic_type (TestSyntheticType): Method for generating synthetic test data.
```

Comment:
I don't know if it makes sense to expose these parameters to the 3d datasets. The anomalous augmentations are only applied to the images, not to the 3d point cloud data.

Reply:
thanks @djdameln, we removed it from this dataset. please let us know if there are more.